overlord/servicestate/quota_handlers.go

```diff
@@ -416,8 +416,8 @@ func quotaRemove(st *state.State, action QuotaControlAction, allGrps map[string]
 		return nil, nil, false, fmt.Errorf("internal error, AddSnaps option cannot be used with remove action")
 	}
 
-	if action.ResourceLimits.Memory != nil {
-		return nil, nil, false, fmt.Errorf("internal error, MemoryLimit option cannot be used with remove action")
+	if !action.ResourceLimits.IsZero() {
+		return nil, nil, false, fmt.Errorf("internal error, quota limit options cannot be used with remove action")
```

Comment:
I looked at these changes in #12093

Reply:
Yea you are right, I accidently included that PR, i'll just remove it